### PR TITLE
Improve NAT gateway logs

### PR DESCRIPTION
**Pull request by vincepri**
Signed-off-by: Vince Prignano <vince@vincepri.com>

**What this PR does / why we need it**:
General NAT gateway logs improvements.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
None

**Special notes for your reviewer**:
None

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: cloud/aws/services/ec2/natgateways_test.go [hunk 1]
```diff
@@ -290,13 +276,14 @@ func TestReconcileNatGateways(t *testing.T) {
...
 			expect: func(m *mock_ec2iface.MockEC2API) {
-
 				m.EXPECT().
-					DescribeNatGatewaysPages(gomock.Any(), gomock.Any()).Times(1)
+					DescribeNatGatewaysPages(gomock.Any(), gomock.Any()).
+					Return(nil).
+					Times(1)
 
-				m.EXPECT().AllocateAddress(gomock.Any()).Times(0)
+				// m.EXPECT().AllocateAddress(gomock.Any()).Times(0)
```
Comment (chuckha) [inline cloud/aws/services/ec2/natgateways_test.go]: Any particular reason you're commenting this out instead of deleting it?

Comment (chuckha) [inline cloud/aws/services/ec2/natgateways_test.go]: any reason why this is commented instead of deleted?

**Comment by vincepri** [comment]
@chuckha @detiber Fixed, this should be good to go

**Pull request merged**